tests/tests/blocks.rs

```diff
@@ -155,53 +153,27 @@ async fn produce_block_manually() {
 async fn produce_block_negative() {
     let db = Database::default();
 
-    let srv = FuelService::from_database(db, Config::local_node())
-        .await
-        .unwrap();
+    let config = Config {
+        debug: false,
+        ..Config::local_node()
+    };
+    let srv = FuelService::from_database(db, config).await.unwrap();
 
     let client = FuelClient::from(srv.bound_address);
 
     let new_height = client.produce_blocks(5, None).await;
 
     assert_eq!(
-        "Response errors; Manual Blocks must be enabled to use this endpoint",
+        "Response errors; `debug` must be enabled to use this endpoint",
         new_height.err().unwrap().to_string()
     );
-
-    let tx = Transaction::default_test_tx();
```

Comment:
Was this removed because it wasn't relevant to testing the functionality of the feature flag?

Reply:
It was not relevant to testing, plus it used `utxo_validation = false` which is not possible anymore with `debug = false`.